### PR TITLE
Re-cast Google Cloud Storage Tutorial as reference page

### PR DESCRIPTION
**Pull request by m57lyra**
## Description

Make Google Cloud Storage page a reference page that is not Hive-centric.

## Release notes

(x) This is not user-visible or docs only and no release notes are required.
( ) Release notes are required, please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: docs/src/main/sphinx/connector/hive-gcs-tutorial.rst [hunk 1]
```diff
@@ -1,108 +1,102 @@
...
-
-Ensure access to GCS
-^^^^^^^^^^^^^^^^^^^^
-
-The :doc:`hive` can access
+Object storage connectors can access
 `Google Cloud Storage <https://cloud.google.com/storage/>`_ data using the
-`Cloud Storage connector <https://cloud.google.com/dataproc/docs/concepts/connectors/cloud-storage>`_.
+`Cloud Storage connector
+<https://cloud.google.com/dataproc/docs/concepts/connectors/cloud-storage>`_.
```
[the remark this answers is not in the document]
Comment (m57lyra) [reply docs/src/main/sphinx/connector/hive-gcs-tutorial.rst:7]: As discussed, it is existing technical content. I think this needs a technical re-visit after, as I believe that Hive is no longer the only supported connector, and it is unclear where the GCS connector comes into play. The only place it is really mentioned is in the POSIX perms section. Is it where folks get the gs:// URI prefix? It's unclear.

File: docs/src/main/sphinx/connector/hive-gcs-tutorial.rst [hunk 2]
```diff
@@ -1,154 +1,148 @@
...
-to perform POSIX permission checks.
-Configuring GCS access for Hive is outside the scope of this tutorial, but there
-are some excellent guides online:
+If Trino and the HMS run as different user accounts, the HMS denies access to
+the Trino user account. You must do one of the following to allow access:
 
-* `Google: Installing the Cloud Storage connector <https://cloud.google.com/dataproc/docs/concepts/connectors/install-storage-connector>`_
-* `HortonWorks: Working with Google Cloud Storage <https://docs.hortonworks.com/HDPDocuments/HDP3/HDP-3.1.0/bk_cloud-data-access/content/gcp-get-started.html>`_
-* `Cloudera: Configuring Google Cloud Storage Connectivity <https://www.cloudera.com/documentation/enterprise/latest/topics/admin_gcs_config.html>`_
+* Run Trino service and Hive service as the same user.
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/hive-gcs-tutorial.rst:66]: HMS

File: docs/src/main/sphinx/connector/hive-gcs-tutorial.rst [hunk 3]
```diff
@@ -1,154 +1,148 @@
...
+* Configure Google Cloud Storage with the ``fs.gs.reported.permissions``
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/hive-gcs-tutorial.rst:67]: where?

**Review by mosabua — CHANGES_REQUESTED** [review]
@m57lyra just a few more notes from a bit of a closer look. At this stage I think we need to work on this together. Maybe even swallow the fact that this has to include technical corrections of the

**Review by electrum — APPROVED** [review]

**Review by mosabua — APPROVED** [review]
Looks good now.

**Pull request merged**